forest/shared/tests/macros_test.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019-2023 ChainSafe Systems
```

Comment:
Could we do it without any `sleep`s or keeping them to the bare minimum? This adds around 1s, but it doesn't have to unless I miss something.

Reply:
fixed